controllers/cron_controller.py

```diff
@@ -444,3 +445,12 @@ def get(self, event_key):
         EventManipulator.createOrUpdate(event)
 
         self.response.out.write(event.district_points)
+
+
+class UpcomingNotificationDo(webapp.RequestHandler):
+    """
+    Sends out notifications for upcoming matches
+    """
+    def get(self):
+        live_events = EventHelper.getEventsWithinADay()
+        NotificationHelper.send_upcoming(live_events)
```

Comment:
I actually think it's better to pass the events as parameters. I tried to keep all the methods in NotificationHelper dumb as to which models they sent data for. That way, we can send out mock notifications of that type.